Sources/Sentry/SentrySessionReplay.m

```diff
@@ -112,7 +118,12 @@ - (void)stop
 
 - (void)captureReplayForEvent:(SentryEvent *)event;
 {
-    if (_isFullSession || !_isRunning) {
+    if (!_isRunning) {
+        return;
+    }
+
+    if (_isFullSession) {
```

Comment:
FYI: I reordered this with the if-check below (line 130), I think JS sdk only tags error/crashed events with replay_id but not messages

Reply:
should we tag messages though? @billyvg 
We don't need to trigger sampling, but if the recording is already ongoing, I imagine it's helpful to link any events captured with it.

Reply:
We should fix the DSC context, and then this won't be required anymore; replay ID will be shared in every type of event after a session mode is started.

Reply:
I don't think we need to tag messages